### PR TITLE
chore: support debug flag for error logging

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates-to: keymanapp/keyman.com#494
Depends-on: keymanapp/shared-sites#38

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ## START STANDARD SITE BUILD SCRIPT INCLUDE
 readonly THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 readonly BOOTSTRAP="$(dirname "$THIS_SCRIPT")/resources/bootstrap.inc.sh"
-readonly BOOTSTRAP_VERSION=v0.10
+readonly BOOTSTRAP_VERSION=chore/v0.11
```
Comment (mcdurdin) [inline build.sh:5]:
```suggestion
readonly BOOTSTRAP_VERSION=v0.11
```

File: developer/language/guide/virtual-keys.md [hunk 2]
```diff
@@ -34,9 +34,7 @@ different ways:
 With a _character code_, in a _positional_ keyboard.
 
 : The key definition references the value on the key cap of a US English
-  hardware keyboard. Caps Lock and Shift are accounted for (see
-  [issue #5460](https://github.com/keymanapp/keyman/issues/5460#issuecomment-966602098)
-  for how to use this correctly with KeymanWeb, using [`&CasedKeys`](#cased-keys)).
+  hardware keyboard. Caps Lock and Shift are accounted for.
```
Comment (darcywong00) [inline developer/language/guide/virtual-keys.md:37]: Is this for a separate PR?

**Review by darcywong00 — APPROVED** [review]
Lgtm

File: developer/language/guide/virtual-keys.md (hunk 2 above)
Comment (mcdurdin) [reply developer/language/guide/virtual-keys.md:37]: yep good catch

**Pull request merged**